src/main/java/com/conveyal/datatools/manager/extensions/ExternalFeed.java

```diff
@@ -0,0 +1,19 @@
+package com.conveyal.datatools.manager.extensions;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * Interface for external feeds that are to be imported into the data manager. The few methods here allow us to
+ * compare feeds with another and get the common fields that should be mapped to internal feed sources.
+ */
+
+public interface ExternalFeed {
+
+    String getFeedUrl();
+
+    String getId();
+
+    String getIdKey();
```

Comment:
What is an Id and IdKey? Are we asking external modules to provide unique IDs for use inside our system?

Reply:
Yes, that's correct.  Is there another way for us to ensure that any updates from external indexes (e.g., to feedURL, name, or other props) make there way back to us (besides the external source's unique IDs)?